### ISSUE TITLE
Update to SOFA version 13

### ISSUE THREAD

**Issue by mhvk** [liberfa/erfa#45]
As noted by @rmathar (https://github.com/liberfa/erfa/issues/44#issuecomment-298229259), SOFA version 13 is out, and we should update ERFA to it, since it solves #40 and #41. It is also a good opportunity to finally get in #42...

**Comment by bsipocz** [comment]
@mhvk  @eteq - Is there any chance to get a new erfa version out to bundle with astropy v2.0?

**Comment by mhvk** [comment]
We should -- but I fear I won't be able to do it...

**Comment by eteq** [comment]
@bsipocz @mhvk - we could pull it into master today (and pull it into Astropy), *but* I don't think I can do the full release.  So it depends on whether we want a "pre-released" version in Astropy.  (We'd still have 2 weeks to iron out any changes, since those could be post feature-freeze but pre-Astropy release).  I don't see that as a new "feature".

@astrofrog or @taldcroft might want to weigh in too, given that it's an Astropy procedural question

**Comment by bsipocz** [comment]
I feel that your proposal is fine for feature freeze, given if you can do a full release before we have final v2.0. (Updating externals doesn't seem to be a new feature to me, but erfa is critical enough that we need the RC testing for it I think.)

**Comment by mhvk** [comment]
Agreed with @bsipocz - pre-release should be fine. I think it would be very good to have even just the erfa version exposed in astropy 2.0.
